### ISSUE TITLE
Sensitive Information Exposed

### ISSUE THREAD

**Issue by hackelite01** [Fsk196/BlogApp#1]
Hi,

I noticed that the file `SECRETKEY.txt` in this repository contains sensitive information such as API keys and project IDs. Exposing these details in a public repository is not secure as it could lead to unauthorized access to your services and data.

**Suggested Action:**
- Remove the `SECRETKEY.txt` file from the repository immediately.
- Use environment variables to manage sensitive information securely.
- Regenerate any exposed keys and update your environment configurations accordingly.

**Potential Risks:**
- Unauthorized access to your services.
- Data breaches and misuse of your resources.
- Compromise of your project's integrity.

For more information on securely managing sensitive information, refer to the [GitHub documentation on secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets).

Thank you.

**Comment by Fsk196** [comment]
Hey thankyou so much for noticing that as i had forgot to remove that :)

**Comment by hackelite01** [comment]
yeah, but simply deleting the file will not secure the repository, as the keys can still be found in the commit history. I suggest:

- Make the repository private to secure it.
- Alternatively, create a new repository with the same code base, ensuring that the sensitive information is not included in the commit history.
